docs/docs/tutorials/eth.md

```diff
@@ -0,0 +1,375 @@
+---
+sidebar_position: 3
+sidebar_label: 'Using web3-eth package'
+---
+
+# Getting Started with `eth` Package
+
+## Introduction
+
+The `web3-eth` package provides a set of powerful functionalities to interact with the Ethereum blockchain and smart contracts. In this tutorial, we will guide you through the basics of using the `web3-eth` package of web3.js version 4. We will be using TypeScript throughout the examples.
+
+## Overview
+
+Here is a high-level overview of the steps we will be taking in this tutorial:
+
+1. Setting up the Environment
+2. Create a new project directory and initialize a new Node.js project.
+3. Set up web3.js and connect to the Ganache network
+4. Interact with the Ethereum blockchain using web3.js.
+
+## Step 1: Setting up the Environment
+
+Before we start writing and deploying our contract, we need to set up our environment. For that, we need to install the following:
+
+1. Ganache - Ganache is a personal blockchain for Ethereum development that allows you to see how your smart contracts function in real-world scenarios. You can download it from http://truffleframework.com/ganache
+2. Node.js - Node.js is a JavaScript runtime environment that allows you to run JavaScript on the server-side. You can download it from https://nodejs.org/en/download/
+3. npm - Node Package Manager is used to publish and install packages to and from the public npm registry or a private npm registry. Here is how to install it https://docs.npmjs.com/downloading-and-installing-node-js-and-npm. (Alternatively, you can use yarn instead of npm https://classic.yarnpkg.com/lang/en/docs/getting-started/)
+
+## Step 2: Create a new project directory and initialize a new Node.js project
+
+First, create a new project directory for your project and navigate into it:
+
+```
+mkdir web3-eth-tutorial
+cd web3-eth-turorial
+```
+
+Next, initialize a new Node.js project using npm:
+
+```
+npm init -y
+```
+
+This will create a new `package.json` file in your project directory.
+
+```
+npm install typescript
+npm install --save @types/node
+```
+
+This will install typescript for our project and install the types for node.
+
+## Step 3: Set up web3.js and connect to the Ganache network
+
+In this step, we will set up the web3.js library and connect to the Ganache network. So, be sure to run Ganache if you did not already did.
+
+First, install the `web3` package using npm:
+
+```
+npm install web3@4.0.1-rc.1
+```
+
+Note that we are installing the latest version of 4.x, at the time of this tutorial writing. You can check the latest version at https://www.npmjs.com/package/web3?activeTab=versions
+
+Next, create a new file called `index.js` in your project directory and add the following code to it:
+
+```javascript
```

Comment:
I think we need to have consistency. I mean, it will be good to have all code written using Typescript. What do you think?

Reply:
fixed, thanks for the catch, it was a mistake.